pkgs/development/python-modules/pyxdg/default.nix

```diff
@@ -15,6 +16,13 @@ buildPythonPackage rec {
   # error: invalid command 'test'
   doCheck = false;
 
+  patches = [ 
+    (fetchpatch {
+      url = "https://gitlab.freedesktop.org/xdg/pyxdg/-/merge_requests/5.patch";
```

Comment:
please use `https://gitlab.freedesktop.org/xdg/pyxdg/-/commit/78405aaa34463db2c6f33ca28ae2293dd3bb1e91.patch` instead, and add a comment above to `https://gitlab.freedesktop.org/xdg/pyxdg/-/merge_requests/5`.

The hash shouldn't break if upstream asks you to do some changes.

Reply:
Fixed